src/python/zigopt/optimize/sources/categorical.py

```diff
@@ -214,12 +213,12 @@ def default_limit(self, limit):
       return 2
     return 1
 
-  """
-  For this optimization source, the categorical hyperparameters are computed, stored and used in future computations.
-  This is in contrast to the structure of the old lion object.
-  """
-
   def get_hyperparameters(self, optimization_args):
+    """
+        For this optimization source, the categorical hyperparameters are computed,
+        stored and used in future computations.
+        This is in contrast to the structure of the old lion object.
+        """
```

Comment:
@malkomes I wasn't sure if it was appropriate to move this doc here

Reply:
seems fine to me, but the second sentence which references "the old lion object" can be removed.